substrate/cli/src/informant.rs

```diff
@@ -57,6 +60,14 @@ pub fn start<C>(service: &Service<C>, exit: ::exit_future::Exit, handle: TaskExe
 		} else {
 			warn!("Error getting best block information");
 		}
+
+		// get cpu usage and memory usage
+		sys.refresh_system();
+		let memory_usage = sys.get_used_memory() as f64 / sys.get_total_memory() as f64;
+		let procs = sys.get_processor_list();
+		let cpu_usage = procs[0].get_cpu_usage();
+		telemetry!("system.usage"; "memory" => memory_usage, "cpu" => cpu_usage);
```

Comment:
Yeah, I was wondering whether it's better to put the logic in polkadot instead of in the informat, then we can exposing some interface to this underlying informat, just like what we did for "network", "client". I'm still learning which is the best way to split the responsibilities between these two.

Reply:
it's fine to do in informant, but i would rather fold it into the `system.interval` telemetry report to minimise chatter and keep the reporting frequency down.